packages/components/forms/src/checkbox/Checkbox.tsx

```diff
@@ -29,7 +29,7 @@ const _Checkbox = (props: CheckboxProps, ref: React.Ref<HTMLInputElement>) => {
     name,
   });
 
-  const formProps = useFormControl({
+  const { isReadOnly, ...formProps } = useFormControl({
```

Comment:
Had to do this change on `Checkbox`, `Radio` and `Switch` as they can't have the readonly state.
And this was giving warnings on formControl.

Reply:
Where is `isReadOnly` being used then? 
eslint is not complaining that this is not being used? 👀

Reply:
It isn't used on the Checkbox at all, and since we use the rest operator here, if we use the `formProps` variable, eslint understand that we are only removing that key from the object.

Reply:
#TIL 
Could you please comment on that line that you are just removing it from the object?

At least, I won't remember this in 2 weeks and then I will ask myself again "What is happening here?"